types/office-js-preview/index.d.ts

```diff
@@ -10564,6 +10564,56 @@ declare namespace Office {
          * @beta
          */
         getInitializationContextAsync(callback?: (asyncResult: Office.AsyncResult<string>) => void): void;
+        /**
+         * Asynchronously gets the ID of a saved item.
+         *
+         * When invoked, this method returns the item ID via the callback method.
+         * 
+         * **Note**: If your add-in calls `getItemIdAsync` on an item in compose mode in order to get an `itemId` to use with EWS or the REST API,
+         * be aware that when Outlook is in cached mode, it may take some time before the item is actually synced to the server.
```

Comment:
"some time" is too vague. I think user the user would appreciate a rough order of magnitude for this duration (seconds, minutes, hours?)

Reply:
I'll follow up on this with the devs but if anything this will be updated in a future PR.